src/lib/rest/rest.cpp

```diff
@@ -1779,10 +1773,11 @@ void restInit
   const char* key  = _httpsKey;
   const char* cert = _httpsCertificate;
 
+  serviceVectorsSet(_getServiceV, _putServiceV, _postServiceV, _patchServiceV, _deleteServiceV, _optionsServiceV, _restBadVerbV);
+
   port             = _port;
-  restServiceV     = _restServiceV;
   ipVersionUsed    = _ipVersion;
-  serveFunction    = (_serveFunction != NULL)? _serveFunction : serve;
+  serveFunction    = (_serveFunction != NULL)? _serveFunction : orionServe;
```

Comment:
This is old stuff from when orion was both ContextBroker and IotAgent and ConfiguratrionManager at the same time.
So, yes, it definitely could be removed, but that is another refactoring. restInit() will change completely etc, etc.
Not for this PR

Reply:
Ok. I'd suggest to mark it with a `// FIXME PR: to be moved within #3109 refactor task` or a similar wording.

Reply:
This has nothing to do with the current refactoring.
I could mark it with a FIXME saying something like "this might be refactored, or not, in the future" ... Really don't see the need of it though.
What **would** help here if we really want to make this change, is a new issue about it

Reply:
Name changed from `orionServe` to `orion::requestServe` in 02d46a83f

Reply:
Next refactoring on my list is exactly this - move RestService vec tors into rest lib. No FIXME needed ... Right?

Reply:
Current situation is that there is an parameter in a function that is never used. I think that is a "weird" situation that worth the one-line of effort ;) that a FIXME PR require.

But you have more neurons on this than me right now, so as you prefer...

Reply:
Sure, I can fix that. Almost easier than putting it in the issue ... :)

Reply:
Fixed in 124c24abf (the variable serveFunction is removed as well - which makes us have to modify images ...)

Reply:
Images modified in eaa194d415ce4048348d1caea2962c82762dff4f